03/app.js

```diff
@@ -1,3 +1,11 @@
 export default function randomNumber(min, max) {
+    if (typeof min !== "number") {
+        throw new Error('"min" is not a number');
+    }
 
-}
\ No newline at end of file
+    if (typeof max !== "number" && !Number.isNaN("number")) {
```

Comment:
Skoro tutaj sprawdzasz czy nie jest to `NaN` to dlaczego nie robisz tego wyże tj. dla `min`? :P